frontend/src/pages/Interventions/ActivitiesComponent.tsx

```diff
@@ -61,9 +61,32 @@ const ActivityComponent: FunctionComponent<ActivityComponentProp> = ({
           imageUrl={activities.a3.url}
           answer={activities.a3.correctAnswer}
           updateIntervention={updateIntervention}
+          key={0}
```

Comment:
This key forces react to rerender, which resets the state of the `ThirdActivity` component. If this is not there, the state does not reset, and it looks really weird.

Reply:
this is interesting